CHANGELOG.md

```diff
@@ -23,13 +23,21 @@ custom checks that rely on the former versions please make necessary amends:
 
 Please refer to the [Integrations Core Checks 5.32.0 tag](https://github.com/DataDog/integrations-core/releases/tag/5.32.0) for the list of changes on the core checks.
 
-Please refer to the [Trace Agent 6.9.0 tag](https://github.com/DataDog/datadog-trace-agent/releases/tag/6.10.0) for more information on the Trace Agent.
-
-Please refer to the [Process Agent 6.9.0 tag](https://github.com/DataDog/datadog-process-agent/releases/tag/6.10.0) for more information on the Process Agent.
+Please refer to the [Process Agent 6.10.0 tag](https://github.com/DataDog/datadog-process-agent/releases/tag/6.10.0) for more information on the Process Agent.
 
 * [SECURITY] pyyaml: allow patching of unsafe operations. See [#3808][]
+* [BUGFIX] Trace: pkg/trace/api: allow empty Type in normalizer. See [datadog-agent-3058](https://github.com/DataDog/datadog-agent/pull/3058) 
```

Comment:
This needs to be removed. It was both introduced and fixed within the development of 6.10. Wasn’t there in 6.9